movieapp/src/components/Button/Button.tsx

```diff
@@ -1,11 +1,18 @@
 type Button = {
   title: string;
+  type: 'submit' | 'reset' | 'button';
+  onclickAction?: React.MouseEventHandler<HTMLButtonElement>;
+  movieId?: string;
 };
 
-export function Button({ title = 'Button title' }: Button): JSX.Element {
+export function Button(props: Button): JSX.Element {
   return (
     <>
-      <button type="button">{title}</button>
+      <button
+          type={props.type}
+          onClick={props.onclickAction}>
```

Comment:
better to pass as: 

onClick={onClick}

without long namings

Reply:
fixed